pkgs/development/python-modules/mergedb/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, pkgs, fetchPypi, setuptools, pyyaml, colorama, jinja2, setuptools-scm, git, python3Packages }:
```

One input per line makes the diffs in the future easier to read.

```suggestion
{ lib, 
, buildPythonPackage
, colorama
, fetchPypi
, jinja2
, pytestCheckHook
, pythonOlder
, pyyaml
, setuptools-scm
}:
```